modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -742,6 +743,33 @@
 				    @warning_ignore("unreachable_code")
 				    print("unreachable")
 				[/codeblock]
+				See also [annotation @warning_ignore_start] and [annotation @warning_ignore_restore].
+			</description>
+		</annotation>
+		<annotation name="@warning_ignore_restore" qualifiers="vararg">
+			<return type="void" />
+			<param index="0" name="warning" type="String" />
+			<description>
+				Restores ignoring the listed warning types after [annotation @warning_ignore_start] to the state specified in the Project Settings. This annotation is optional, you are not required to use it if you want to ignore some warning type until the end of the file.
+				[b]Note:[/b] Unlike most other annotations, arguments of the [annotation @warning_ignore_restore] annotation must be string literals (constant expressions are not supported).
+			</description>
+		</annotation>
+		<annotation name="@warning_ignore_start" qualifiers="vararg">
+			<return type="void" />
+			<param index="0" name="warning" type="String" />
+			<description>
+				Enables ignoring the listed warning types until the end of the file or until the parser encounters [annotation @warning_ignore_restore] with this warning type.
```

Comment:
This reads like a double negation. From the outside user's perspective the "parser encountering something" is a bit unusual to say, but I have no suggestions at the moment.
```suggestion
				Ignores the listed warning types until the end of the file or until the parser encounters [annotation @warning_ignore_restore] with the given warning types.
```